content/docs/howto/negate-a-disjunction/en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Negate a disjunction
+tags:
+- validation
+- language
+- commented cue
+authors:
+- hansbogert
+relatedIssues:
+- https://github.com/cue-lang/cue/discussions/3229
+- https://github.com/cue-lang/docs-and-content/issues/160
+toc_hide: true
+---
+
+This [Commented CUE]({{< relref "docs/howto#commented-cue-guides" >}})
+demonstrates how to negate a disjunction.
+
+{{{with code "en" "example"}}}
+! exec cue vet example.cue
+cmp stderr out
+-- example.cue --
+package example
+
+// Define a disjunction
+#oneTwoThree: "one" | "two" | "three"
+
+// Define a concrete value
+four: "four"
+// define a hidden helper value which makes sure
+// that `four` is not #oneTwoThree by comparing
```

Comment:
(Another reason to unaligned the `"four" value from a field of the same name!)

Let's spell out what "X is not Y" means here, slightly. Something like...

```
// These hidden fields ensure that the data fields they reference do not contain values permitted by the #oneTwoThree disjunction.
``` 
(word-wrapped!)